source/_components/sensor.ambient_station.markdown

```diff
@@ -0,0 +1,85 @@
+---
+layout: page
+title: "Ambient Weather Station Sensor"
+description: "How to integrate Ambient Weather station within Home Assistant."
+date: 2018-11-15 08:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Weather
+featured: false
+ha_release: "0.83"
+ha_iot_class: "Cloud Polling"
+---
+
+The `Ambient Weather Station` platform uses the [Ambient Weather](https://ambientweather.net)  
+web API to retrieve weather data from your personal weather station (PWS).
+
+You need to register your compatible PWS using its MAC address and then request both an API key and an Application key.  The key requests can be found
+under `My Account`.  Requesting an application key requires emailing the ambient weather support team.
+
+To add your Ambient Weather PWS to your Home Assistant installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+sensor:
+  - platform: ambient_station
+    api_key: YOUR_API_KEY
+    app_key: YOUR_APP_KEY
+    monitored_conditions:
+      - tempf
+      - humidity
+      - windspeedmph
+```
+
+Configuration variables:
```

Comment:
Use the new configuration tag.

https://developers.home-assistant.io/docs/en/documentation_create_page.html#configuration